integration/env/env.go

```diff
@@ -71,13 +79,38 @@ func CircleSHA() string {
 }
 
 func ClusterID() string {
-	return fmt.Sprintf("ci-awsop-%s-%s", TestedVersion(), CircleSHA()[0:5])
+	var parts []string
+
+	parts = append(parts, "awsci")
```

Comment:
Why changing the previous convention? We are used to look for resources named after `ci-*`.

Reply:
My fear is it gets too long and I wanted to be smart to safe a bit length. I also do not like the `awsop` thing. I think it is possible people get used to `awsci` instead of `ci`. Should be doable for most of us. WDYT?

Reply:
IMO the `aws` prefix doesn't add anything, all the resources live in aws anyway and they are isolated and only live during the build, no possible collisions on host cluster or things like that. The previous `awsop` prevented collisions with resources created for aws-terraform (now deprecated), `op` had a reason there but arguably aws could be removed on aws-terraform case too.

So, what about leaving only `ci`, something like `ci-wip-3cc75-5e958`?

Reply:
I like this.

Reply:
Done.

Reply:
+1 on keeping the `ci` prefix it's the useful part.